web/src/message_view_header.js

```diff
@@ -31,6 +32,7 @@ function make_message_view_header(filter) {
         };
     }
     message_view_header.title = filter.get_title();
+    message_view_header.is_guest = page_params.is_guest;
```

Comment:
This function has some weird naming conventions; we probably should call the `message_view_header` variable `context`, and call the function `get_message_view_header_context`, since I think it's just used as context for rendering a template...

Reply:
Renamed the variables `message_view_header` and `message_view_header_data` with `context`. We also rename the function `make_message_view_header` with `get_message_view_header_context`. Please note that I have also renamed similar variables outside the function ( and throughout the file) as well.